package.json

```diff
@@ -179,5 +180,8 @@
   "repository": {
     "type": "git",
     "url": "git+https://github.com/bigbinary/neeto-editor.git"
+  },
+  "dependencies": {
+    "react-i18next": "^12.1.5"
   }
```

Comment:
1. Move `react-i18next` as a dev dependency instead of a dependency.
2. Add both `i18next` and `react-i18next` packages to peer dependencies.
3. The version of both of these packages should match the version in [package-common.json](https://github.com/bigbinary/neeto-commons-backend/blob/main/lib/neeto_commons_backend/common_files/package-common.json) file.
4. Update the documentation regarding installation of neetoEditor. It needs to mention that these packages should be installed as peer dependencies.